stripe/_http_client.py

```diff
@@ -502,7 +502,7 @@ async def _request_with_retries_internal_async(
         *,
         _usage: Optional[List[str]] = None
     ) -> Tuple[Any, int, Mapping[str, str]]:
-        self._add_telemetry_header(headers)
```

Comment:
this was a bug. This meant that telemetry wasn't being recorded at all for async requests

Reply:
Nice catch! Looks like there's some tests that need updating after fixing this bug.

Reply:
Fixed in [Fix tests](https://github.com/stripe/stripe-python/pull/1266/commits/45f16ebc99b30b9d07e453601ef5d410c1c845ba)